pkg/controller.v1/tensorflow/controller.go

```diff
@@ -65,6 +65,10 @@ const (
 
 	TFCleanPodStatusLabel = "arena.kubeflow.org/clean-pod-status"
 	TFCleanStatusDone     = "done"
+
+	TFPodGroupSettingLabel     = "pod-group.scheduling.sigs.k8s.io/name"
+	TFGangJobRunningAnnotation = "arena.kubeflow.org/gang-running"
```

Comment:
I suggest renaming `arena.kubeflow.org/gang-running` to `arena.kubeflow.org/pod-running`.